package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "no-hassle",
-  "version": "2.10.0",
+  "version": "3.0.0",
```

Comment:
I'll have to `yarn release` after this is approved? Not sure how we release this.